### ISSUE TITLE
Document how to run awx-operator locally for development 

### ISSUE THREAD

**Issue by TheRealHaoLiu** [ansible/tower-operator#1364]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that AWX Operator is open source software provided for free and that I might not receive a timely response.

### Feature Summary

operator-sdk and ansible-operator (and makefile for this repo) have `make run` target that runs the awx-operator on developers laptop.

also ansible-operator run should also add `--zap-encoder=console` so that the ansible playbook output display in a pretty way

**Comment by shanemcd** [comment]
We already have docs at https://github.com/ansible/awx-operator/blob/devel/docs/debugging.md.

**Comment by rooftopcellist** [comment]
Agreed though, docs around using the `make run` or `make deploy` target would be useful.  

These docs could use make-over too:
* https://github.com/ansible/awx-operator/blob/devel/CONTRIBUTING.md

**Comment by bobby0724** [comment]
For local development I use this one and is pretty good[
https://github.com/kurokobo/awx-on-k3s
